catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Node.collection.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+/*jshint -W024*/
+define([
+    'backbone',
+    './Node'
+], function (Backbone, Node) {
+
+    return Backbone.Collection.extend({
+        model: Node.Model,
+        comparator: function(model){
+            return model.toJSON().RegistryObjectList.ExtrinsicObject[0].Name;
```

Comment:
Sounds good. @andrewkfiedler that explains why the data was wrapped in a list. 

@clockard in our test cases we always had a list of size 1, with all of our node data in that single member. In production, is that not guaranteed?